05.wc/wc.rb

```diff
@@ -0,0 +1,84 @@
+# frozen_string_literal: true
+
+require 'optparse'
+
+LINES_WIDTH = 7
+WORDS_AND_CHARACTERS_WIDTH = 8
+
+def main
+  options = parse_options
+
+  if ARGV.empty?
+    print_statistics_from_standard_input(options)
+  else
+    print_statistics_and_total_from_argv(options)
```

Comment:
totalは場合によって出力されない場合もあります。
ここも具体的にしすぎると実際の挙動と矛盾することがあるので、totalもstatisticsの1つ、と見なすといいと思います。

```suggestion
    print_statistics_from_argv(options)
```

Reply:
ありがとうございます！
ご提案を基に修正させていただきました🙇